tck/bridge-tck-common/src/main/java/com/liferay/faces/bridge/tck/common/util/BridgeTCKUtil.java

```diff
@@ -26,15 +26,13 @@
 public class BridgeTCKUtil {
 
 	public static boolean isHeaderOrRenderPhase(Bridge.PortletPhase portletPhase) {
-		return (Bridge.PortletPhase.HEADER_PHASE.equals(portletPhase) ||
-			Bridge.PortletPhase.RENDER_PHASE.equals(portletPhase));
+		return Bridge.PortletPhase.HEADER_PHASE.equals(portletPhase);
```

Comment:
@stiemannkj1 Since running the JSF lifecycle in the HEADER_PHASE is now in the Spec, in the code review I'm deleting the BridgeTCKUtil and am introducing diffs in the tests. In other words, for Portlet 3.0 branches the TCK tests will look like:
```
if (Bridge.PortletPhase.HEADER_PHASE.equals(portletPhase) {
```
But in Portlet 2.0, the TCK tests will look like:
```
if (Bridge.PortletPhase.RENDER_PHASE.equals(portletPhase) {
```